assets/learning-path-navigation.ts

```diff
@@ -0,0 +1,104 @@
+import { learning_paths as paths } from "json-collector";
+
+export function init() {
+  const currentLocation = window.location.href;
+  const params = new URLSearchParams(document.location.search);
+  const currentLearningPath = params.get("learning_path");
+  let currentPathData;
+
+  if (currentLearningPath !== null) {
+    for (const path in paths) {
+      if (paths[path]["uid"] === currentLearningPath) {
+        currentPathData = paths[path];
+      }
+    }
+
+    // Add learning path to breadcrumb list
+    const firstLearningBreadcrumb = document.getElementById(
+      "firstLearningBreadcrumb"
+    );
+    if (firstLearningBreadcrumb) {
+      firstLearningBreadcrumb.insertAdjacentHTML(
+        "afterend",
+        `<li id="firstLearningBreadcrumb">
+            <a href="${currentPathData["path"]}" class="DocsMarkdown--link">
+              <span class="DocsMarkdown--link-content">${currentPathData["title"]}</span></a>
+          </li>`
+      );
+    }
+
+    // Update final next link to point to the next module
+    const nextModuleLink = document.getElementById("nextModuleLink");
+    if (nextModuleLink && currentPathData !== undefined) {
```

Comment:
```suggestion
    if (nextModuleLink && currentPathData) {
```
This will also fail if `currentPathData` is undefined.